includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -712,7 +712,7 @@ public function stripe_connect_field( $value, $option_value ) {
 					if ( give_stripe_is_connected() ) :
 						$site_url            = get_site_url();
 						$stripe_user_id      = give_get_option( 'give_stripe_user_id' );
-						$modal_title         = __( '<strong>You are connected! Now this is important: Please now configure your Stripe webhook to finalize the setup.</strong>', 'give' );
+						$modal_title         = '<strong>' . __( 'You are connected! Now this is important: Please now configure your Stripe webhook to finalize the setup.', 'give' ) . '</strong>';
```

Comment:
@bradp can we use `esc_html__`

Reply:
It doesn't make sense to me to pass in HTML for the translators, as we don't want the markup changed.

Reply:
@bradp I agree on this but can we replace `__` with `esc_html__`